en/user_guide/media.rst

```diff
@@ -0,0 +1,57 @@
+Media Management
+================
+
+Open Orchestra allows to manage media in the media library panel
+
+Folder
+------
+
+All files are stocked in folder. It's possible to move in the folder tree to see files by click on the left panel.
```

Comment:
All files are organised inside folders.
possible to browse the folder tree
by clicking